test/scripts/unsubscribe_inactive_people_test.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class UnsubscribeInactivePeopleTest < ActiveSupport::TestCase
+  setup :create_items
+  test 'unsubscribe inactive people' do
+    unsubscribe
+
+    assert_equal UserNotification.all.size, 1
+    assert_equal UserNotification.first.user_id, @user.id
+    assert_includes UserNotification.all, @un
+    assert_raises ActiveRecord::RecordNotFound do
+      UserNotification.find(@un2.id)
+    end
+  end
+
+  private def create_items
+    load './scripts/unsubscribe_inactive_people.rb'
```

Comment:
If this is to be inside this function, I suggest renaming the `create_items` function as it is no longer about just creating items; it also loads the script.